TODO.md

```diff
@@ -27,6 +27,6 @@ server
 - [ ] Add auto-publish/no publish as a parameter for each cog, perhaps `auto-publish-channels = [<chan_ID1>, <chan_ID1>]`
 - [ ] In-game server notifications posted to Discord channels
 - [x] Convert to use `setup.cfg` instead of `setup.py`
-- [ ] Add `old_rates` and `current_rates` to `RatesDiff` so `self.current_rates` could be used instead of `to_embed` 
-- [ ] Allow for `old_rates=old` and `new_rates=new` to be passed into `RatesStatus.to_diff()`  
+- [x] Add `old_rates` and `current_rates` to `RatesDiff` so `self.current_rates` could be used instead of `to_embed` 
+- [x] Allow for `old_rates=old` and `new_rates=new` to be passed into `RatesStatus.to_diff()`  
```

Comment:
I don't know if I actually accomplished what you had intended here

Reply:
I don't know if I correctly expressed what you intended here, because these two TODO's were just me paraphrasing something you suggested.